test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -2858,3 +2858,57 @@ def test_lowering_to_x86(self):
                 node_list,
                 lower=True,
             )
+
+    @skipIfNoX86
+    def test_annotate_mul_tensor(self):
+        class M1(torch.nn.Module):
+            def __init__(self, type):
+                super().__init__()
+                self.type = type
+
+            def forward(self, x, y):
+                if self.type == 0:
+                    return x * y
+                elif self.type == 1:
+                    return x * y.sum(-1)
+                elif self.type == 2:
+                    return x * y.sum()
+                else:
+                    return x * y.sum().item()
+
+        for type in [0, 1, 2, 3]:
```

Comment:
nit: I think we can improve [0, 1, 2, 3] by defining different classes for each test case here

Reply:
Thanks. Updated.